### PR TITLE
UI assets

### PR DESCRIPTION
**Pull request by harsh242**
**What does this PR do?**
Adds UI assets to academy, customizer and landing page

**Description of Task to be completed?**

**How should this be manually tested?**
A clear and concise description of how the changes can be tested.

**Any background context you want to provide?**

**Screenshots(optional)**
@manangouhari here's the code
getting a graphql error while adding images to mdx files
image
image

**Additional context(optional)**
Add any other context if any.

### CODE REVIEW

File: lessons/module/module-00/module-00.mdx [hunk 1]
```diff
@@ -3,6 +3,7 @@ slug: module-0
 type: module
 next: '/tezos/storyline'
 title: Basics of blockchain
+img:../../../src/images/bot1.png
```
Comment (bhaskarSingh) [inline lessons/module/module-00/module-00.mdx:6]: @harsh242  Importing it like a component should fix the error. eg: `import image1 from "../../../src/images/bot1.png";

Comment (manangouhari) [reply lessons/module/module-00/module-00.mdx]: Fix was to add space between `img:` and the image path.

**Comment by manangouhari** [comment]
@harsh242 please do the same for `/academy/module-*` page too.

**Comment by bhaskarSingh** [comment]
@harsh242 You will have to create a new branch from develop & push all the changes from this branch to the newly created branch. Ideally, it's always better to branch out from develop branch to avoid unnecessary merge conflicts. Now that MVP-branch is not mergeable due to auth issue.

**Pull request closed without merging**